src/test/regress/specs/isolation_add_node_vs_copy.spec

```diff
@@ -0,0 +1,72 @@
+# remove one of the nodes for the purpose of the test
+setup
+{	
+	SELECT master_remove_node('localhost', 57638);
+	
+    CREATE TABLE test_reference_table (test_id integer);
+    SELECT create_reference_table('test_reference_table');
+}
+
+# ensure that both nodes exists for the remaining of the isolation tests
+teardown
+{
+	DROP TABLE test_reference_table;
+	SELECT nodename, nodeport, isactive FROM master_add_node('localhost', 57637);
+	SELECT nodename, nodeport, isactive FROM master_add_node('localhost', 57638);
```

Comment:
We could do that, but now the test is independent of its place. Isn't the current structure better?